### PR TITLE
Chore/next upgrade

### PR DESCRIPTION
**Pull request by keraf**
This change updates the following dependencies across the entire project (where applicable):
- Next -> 9.5.3
- React -> 16.13.1
- React DOM -> 16.13.1
- React Native -> 0.63.2

Along with the dependency update, some warnings and other issues that appeared with the update were addressed.

This change also introduces a new ESLint rule to flag anonymous function exports (which causes issues with Next's fast refresh).

### CODE REVIEW

**Comment by slowbackspace** [comment]
upgrading react-native used to be more complicated than just bumping version of the dependencies. I am assuming that's still the case. You can use https://react-native-community.github.io/upgrade-helper/ to see diff between simple hello world apps built on different react-native version. Also it is always good to check if the android/ios build is actually running on a device/emulator.

you can also follow this tutorial (linked in the upgrade helper tool)
https://reactnative.thenativebits.com/courses/upgrading-react-native/upgrade-to-react-native-0.62/

**Comment by keraf** [comment]
Thanks for pointing me to this @slowbackspace. I updated all files except one (this step seems to require a Mac, which I don't have). Could you maybe look at updating `packages/suite-native/ios/TrezorSuite.xcodeproj/project.pbxproj` for me please? The upgrade helper points here for more details: https://github.com/react-native-community/upgrade-support/issues/13

I also haven't tested it on Android yet. Still need to get the environment up and running on my machine. I'll do this during the weekend.

**Comment by slowbackspace** [comment]
Finished updating ios files. At least the build process now runs successfully. The app will ofc crash at startup, but I don't wanna spent more time on this right now (it could be that I miss something during the upgrade or it is just that ios app was not running even before the upgrade).

tldr:
- fixed paths to react-native
- removed old LaunchScreen.xib references
- bumped deploy target version to 11
- applied settings recommended by xcode

**Review by prusnak — APPROVED** [review]
LGTM
